UPGRADING.md

```diff
@@ -259,11 +280,11 @@ the full RPC interface provided as direct function calls. Import the
 the node service as in the following:
 
 ```go
-    node := node.NewDefault() //construct the node object
-    // start and set up the node service
+	node := node.NewDefault() //construct the node object
+	// start and set up the node service
```

Comment:
Since we're in here cleaning anyway
```suggestion
logger := log.NewNopLogger()

// Construct and start up a node with default settings.
node := node.NewDefault(logger)
```